scripts/biomedical/NCBI_Taxonomy/test_data/nodes_enum_expected.mcf

```diff
@@ -0,0 +1,184 @@
+Node:dcid:BiologicalTaxonomicRankEnum
```

Comment:
same as issue as the "division_enum_expected.mcf" file - please fix

Reply:
This file is intended for unit testing purposes only. All known issues have been resolved.